server/chat-commands.js

```diff
@@ -622,20 +622,34 @@ const commands = {
 	'!away': true,
 	idle: 'away',
 	afk: 'away',
-	away(target, room, user) {
-		this.parse('/blockchallenges');
-		this.parse(`/blockpms ${target}`);
+	brb: 'away',
+	away(target, room, user, connection, cmd) {
+		let awayType = toID(cmd);
```

Comment:
Are commands case-sensitive? Maybe we should make them consistently lowercase at some point.

Reply:
I tried `/AWAY` and `/Away` in chat just now in the main chat and both work, so I think this is required currently.

Reply:
Heh, I actually talked about this with bumba and apparently the `toID` _isn't_ redundant, but we agreed to change it so it was.

Reply:
But then again maybe we were wrong. @bumbadadabum said he tested it but I didn't exactly confirm his findings.

Reply:
Yes, we were very wrong. @bumbadadabum `Context#splitCommand` already converts `cmd` to lowercase. `/me` already uses ``this.message.startsWith(`/ME`)``. This can just be fixed directly.